app/src/main/kotlin/com/wire/android/ui/conversation/common/RowItem.kt

```diff
@@ -0,0 +1,25 @@
+package com.wire.android.ui.conversation.common
+
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.RowScope
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.wire.android.ui.common.SurfaceBackgroundWrapper
+import com.wire.android.ui.theme.Dimensions
+
+
+@Composable
+fun RowItem(content: @Composable RowScope.() -> Unit) {
+    SurfaceBackgroundWrapper(modifier = Modifier.padding(0.5.dp)) {
```

Comment:
Is half DP correct?

Also, is there a chance we would reuse this and then we could add to `Dimensions`?

Reply:
in the designs there is 1dp, but somehow it looks weird to me so I'll leave it at 0.5.dp untill design review and will move this dimenstion to dimension.kt file